Library/Homebrew/global.rb

```diff
@@ -62,7 +62,6 @@ module Homebrew
   "-S" => "search",
   "up" => "update",
   "ln" => "link",
-  "instal" => "install", # gem does the same
```

Comment:
Please leave it here. It’s a handy shortcut for people typing too fast: `brew instal git` is thus an alias for `brew install git`.